spa/src/components/account/ResetPassword/ResetPasswordForm.test.js

```diff
@@ -8,24 +8,50 @@ const mockSubmit = jest.fn((email, password) => {
 });
 
 describe('ResetPasswordForm Tests', () => {
+  function getScreen() {
+    return render(<ResetPasswordForm onResetPasswordSubmit={mockSubmit} loading={false} />);
+  }
+
+  it('should have password toggle icon to show/hide value of password if its visibility icon is clicked', () => {
+    getScreen();
+    const password = screen.getByTestId(`reset-pwd-${Input.types.PASSWORD}`);
+    fireEvent.input(password, {
+      target: {
+        value: 'password'
+      }
+    });
+    const toggleIcon = screen.getByTestId('toggle-password');
+    fireEvent.click(toggleIcon);
+    expect(password.getAttribute('type')).toEqual(Input.types.TEXT);
+    fireEvent.click(toggleIcon);
+    expect(password.getAttribute('type')).toEqual(Input.types.PASSWORD);
```

Comment:
Suggestion, could also add a check for the initial:
```suggestion
    const toggleIcon = screen.getByTestId('toggle-password');
    expect(password.getAttribute('type')).toEqual(Input.types.PASSWORD);
    fireEvent.click(toggleIcon);
    expect(password.getAttribute('type')).toEqual(Input.types.TEXT);
    fireEvent.click(toggleIcon);
    expect(password.getAttribute('type')).toEqual(Input.types.PASSWORD);
```

Reply:
done